src/awsHttpHandler/httpHandler.ts

```diff
@@ -0,0 +1,116 @@
+import { hckFetch } from '../index';
+import { requestTimeout } from './requestTimeout';
+import { isEmpty } from './isEmpty';
+
+type LogHandler = (...content: unknown[]) => string;
```

Comment:
Log functions are typically `void`.

Moreover, I think that you should clarify the type of the parameters to be able to safely invoke the function: `...content: unknown[]` is too vague. Remember that the library defines the contract that the callee needs to honour, not the opposite

```suggestion
type LogFn = (message: string, ...params?: unknown[]) => void;
```